<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Yüz Tanıma İşlemleri</string>
  </property>
  <widget class="QPushButton" name="buton1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Kamera Kapat</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buton2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>150</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Veri Seti</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buton4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>419</y>
     <width>150</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Tanıla</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buton3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>150</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Eğit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>260</y>
     <width>47</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Adı</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="kisi_adi">
   <property name="geometry">
    <rect>
     <x>58</x>
     <y>254</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="kisi_id">
   <property name="geometry">
    <rect>
     <x>58</x>
     <y>290</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="kamera_label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>550</width>
     <height>450</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="yazi_alani">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Aşağıdaki Özellikleri Kullanmak İçin 
Önce Kamerayı Açın</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Kamera Aç</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
